package.json

```diff
@@ -121,6 +121,8 @@
   },
   "resolutions": {
     "arbundles/arweave": "^1.13.0",
+    "**/axios": ">=0.28.0",
+    "**/msgpackr": ">=1.10.1",
     "human-crypto-keys/node-forge": "^1.3.1",
     "human-crypto-keys/crypto-key-composer/node-forge": "^1.3.1",
     "@parcel/runtime-js": "2.8.3"
```

Comment:
Tested with plasmo@0.86.3, and removing this line doesn't give me any warnings. Could you test that?